### ISSUE TITLE
Exception in pub build

### ISSUE THREAD

**Issue by scheglov** [dart-lang/pub#1362]
@MikeMitterer
@munificent 
@nex3

Mike, I noticed that there was an exception on pub build execution before pub get.
Natalie would like to get more information about this problem.

See https://github.com/dart-lang/sdk/issues/24960

**Comment by nex3** [comment]
Specifically, I'd like to know what version of `pub` you're using and I'd like to see a `--verbose` trace of the error.

**Comment by MikeMitterer** [comment]
Which project do you mean - I have so many? Is it on the last video I sent you? http://www.mikemitterer.at/fileadmin/_temp_/dart_analyzer.mp4

**Comment by scheglov** [comment]
Yes, in this video at about 18:00.

**Comment by MikeMitterer** [comment]
This happened only on the first try. Then I made the "pub get", it updated (probably) the .packages file and then pub build worked...
MobiAd.REST.UI.MDL includes the webapp_base_ui_md package
```yaml
  webapp_base_ui_mdl:
    path: /Volumes/Daten/DevLocal/DevDart/WebApp.Base.UI.MDL
```

webapp_base_ui_mdl includes the mdl-package:
```yaml
  mdl:
    path: /Volumes/Daten/DevLocal/DevDart/MaterialDesignLite
```
and the mdl-package include the mustache-lib:
```yaml
  mustache: ^0.2.0
```
*This is what changed!* I haven't include the mustache-lib before. I changed this in the mdl-package.
Moved over to  MobiAd.REST.UI.MDL and made the pub build, it crashed, I made the pub get (18:48 vid) it updated the package, loaded the mustache dependencies. From now on pub build worked.
